src/App.js

```diff
@@ -1,7 +1,3 @@
-import './style.css';
+// import App from './components/App';
 
```

Comment:
## Remove unnecessary file
- [ ] This project should only have one `App.js` file which you have rightfully placed in the `components` directory. DO you mind removing this one as it is redundant and of no use?

Reply:
Done, forgot to remove it after I made everything work haha!